apps/sms/js/thread_ui.js

```diff
@@ -158,7 +158,9 @@ var ThreadUI = {
     );
```

Comment:
I don't like all the changes from this file (change from Threads.active to a promise). I feel like I should take it and store it in a local var in beforeEnterThread and just use it afterwards.